calico_node/Makefile

```diff
@@ -1,13 +1,19 @@
 ###############################################################################
-# Versions:
-RELEASE_STREAM?=v2.5
+#
```

Comment:
I think this is a stray `#`?

Reply:
yep! thanks for the Friday afternoon PR review :-)